docs/contribution-guides/3. run-authentication-sample.md

```diff
@@ -0,0 +1,80 @@
+# Contribution Guides
+
+1. **[Get Set up](<./contribution-guides/1. get-set-up.md>)**
+2. **[Build Authentication Server Sample](<./contribution-guides/2. build-authentication-server-sample.md>)**
+3. **[Run Authentication Server Sample](<./contribution-guides/3. run-authentication-server-sample.md>)**
+4. **[Test Your Changes](<./contribution-guides/4. test-your-changes.md>)**
+5. **[Write Unit Tests](<./contribution-guides/5. write-unit-tests.md>)**
+6. **[Submit a PR](<./contribution-guides/6. submit-a-pr.md>)**
+7. **[Publish Your Changes](<./contribution-guides/7. publish-your-changes.md>)**
+
+
+
+## 3. Run Authentication Server Sample
+
+### Prerequisites
+
+- Register a Client and Server (Web API) applications in Azure Active Directory (AAD) as part of [On Behalf Of workflow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow). See instructions below
+- Update the Server (Web API) application with information from the app registrations
+
+### [Set up App Registrations](../deploy/set-up-app-registrations.md)
+
+### Create an Azure Communication Services resource
+
+Follow [Quickstart: Create and manage Communication Services resources](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource?tabs=windows&pivots=platform-azp) to create an Azure Communication Services resource using Azure Portal.
+
+### Create an `appSettings.ts` File
+
+Before running the sample, you will need to replace the values in the  `appSettings.ts`  file:
+
+1. Use the `appSettings.ts.template` file as an template to generate the  `appSettings.ts`  file.
+
+2. Replace `connectionString` and `scopes` for the Communication Services
+
+3. Replace `clientId`, `tenantId` and `clientSecret` for the Azure Active Directory.
+
+   > Note: Values of `clientId`, `tenantId` and `clientSecret` are all from `AuthServer` app registration.
+
+### Generate an Azure AD Token manually
+
+Since the sample does not have a client application, you need to generate Client Azure AD Token manually to make calls to Azure Active Directory protected backend Web APIs in the sample. You will need an access token using client app registration to call the Web API. In order to get the access token manually, please follow steps [here](../test-tools/generate_aad_token_manually.md). 
+
+>  Note: If you are integrating a client application, then please ignore these steps as you could test directly via user signing through client application.
+
+Once you get the `access_token` in the response, you can jump to the next step to start the server and call `http://localhost:5000/api/token` or `https://localhost:5001/api/token` using the  `access_token` .
```

Comment:
I think the port here seems to be of csharp 5000 and 5001, could you please revise this?

Reply:
what do you mean